packages/bits/src/lib/textbox/textbox.component.ts

```diff
@@ -36,10 +36,6 @@ import { NuiFormFieldControl } from "../form-field/public-api";
     ],
     styleUrls: ["./textbox.component.less"],
     encapsulation: ViewEncapsulation.None,
-    host: {
-        "role": "textbox",
-        "[attr.aria-label]": "ariaLabel + ' textbox'",
-    },
 })
 export class TextboxComponent implements ControlValueAccessor, NuiFormFieldControl {
     /**
```

Comment:
Please, remove `@Input() public ariaLabel: string = "";` from textbox. If you don't need ariaLabel anymore.

Reply:
aria label is still used in the template